src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/Common/WebRequestPSCmdlet.Common.cs

```diff
@@ -314,27 +314,26 @@ public virtual string CustomMethod
         public virtual IDictionary Form {get; set;}
 
         /// <summary>
-        /// gets or sets the ContentType property
+        /// Gets or sets the ContentType property.
         /// </summary>
         [Parameter]
         public virtual string ContentType { get; set; }
 
         /// <summary>
-        /// gets or sets the TransferEncoding property
+        /// Gets or sets the TransferEncoding property.
         /// </summary>
         [Parameter]
         [ValidateSet("chunked", "compress", "deflate", "gzip", "identity", IgnoreCase = true)]
         public virtual string TransferEncoding { get; set; }
 
         /// <summary>
-        /// gets or sets the InFile property
+        /// Gets or sets the InFile property.
         /// </summary>
         [Parameter]
         public virtual string InFile { get; set; }
 
         /// <summary>
-        /// keep the original file path after the resolved provider path is
-        /// assigned to InFile
+        /// Geep the original file path after the resolved provider path is assigned to InFile.
```

Comment:
Spelling error: should be 'Keep' instead of 'Geep'.

Reply:
Fixed.